### ISSUE TITLE
Theme causes major headaches

### ISSUE THREAD

**Issue by gazzafm** [civicrm/org.civicrm.shoreditch#221]
Installed this theme with 4.7 Civi and followed the settings. Installed Civicase 5 extension. The layout was all over the place. Panels on top of each other, major spaces between panels, and in some cases text was outside border areas. Disappointing that Civi still has an outdated UI and was hoping for better with Shoreditch, but unfortunately
![capture](https://user-images.githubusercontent.com/22442027/38870547-7a45b716-4291-11e8-918a-8418ebab11e1.JPG)
![capture1](https://user-images.githubusercontent.com/22442027/38870548-7a75872a-4291-11e8-93dc-6d0c182274e2.JPG)

 its not working like some screen shots show.

**Comment by colemanw** [comment]
@gazzafm that's definitely not what it's supposed to look like. I've been using shoreditch for a while and haven't seen that problem. It could be caused by interference from your CMS. What CMS/theme are you using?

**Comment by gazzafm** [comment]
We are using Drupal and using the standard theme that comes with. We havnt installed any other theme.

**Comment by sleewok** [comment]
Question: Within the civicrm resource settings, should be be disabling the civicrm.css? 
![image](https://user-images.githubusercontent.com/304940/43900730-ab49ec80-9bb3-11e8-888f-4ddafc2c5b9e.png)

**Comment by AkA84** [comment]
@sleewok yes, the "legacy mode" (`custom-civicrm.css`) is built on top of the original civicrm.css so you also need that on your site

@colemanw @gazzafm not sure if this is more about civicase than shoreditch. Any update?
